mobile/mysterium/proposals_manager.go

```diff
@@ -83,24 +83,30 @@ func newProposalsManager(
 	repository proposal.Repository,
 	mysteriumAPI mysteriumAPI,
 	qualityFinder qualityFinder,
-	lowerPriceBound, upperPriceBound uint64,
+	lowerTimePriceBound, upperTimePriceBound uint64,
+	lowerGBPriceBound, upperGBPriceBound uint64,
+
 ) *proposalsManager {
 	return &proposalsManager{
-		repository:      repository,
-		mysteriumAPI:    mysteriumAPI,
-		qualityFinder:   qualityFinder,
-		upperPriceBound: upperPriceBound,
-		lowerPriceBound: lowerPriceBound,
+		repository:          repository,
+		mysteriumAPI:        mysteriumAPI,
+		qualityFinder:       qualityFinder,
+		upperTimePriceBound: upperTimePriceBound,
+		lowerTimePriceBound: lowerTimePriceBound,
+		lowerGBPriceBound:   lowerGBPriceBound,
+		upperGBPriceBound:   upperGBPriceBound,
 	}
 }
 
 type proposalsManager struct {
-	repository      proposal.Repository
-	cache           []market.ServiceProposal
-	mysteriumAPI    mysteriumAPI
-	qualityFinder   qualityFinder
-	upperPriceBound uint64
-	lowerPriceBound uint64
+	repository          proposal.Repository
+	cache               []market.ServiceProposal
+	mysteriumAPI        mysteriumAPI
+	qualityFinder       qualityFinder
+	upperTimePriceBound uint64
```

Comment:
I suggest to inject `proposal.Filter` instead of proxying every filter variable

Reply:
done